src/diff/props.js

```diff
@@ -54,13 +58,14 @@ function setStyle(style, key, value) {
  * @param {boolean} isSvg Whether or not this DOM node is an SVG node or not
  */
 function setProperty(dom, name, value, oldValue, isSvg) {
-	name = isSvg
-		? name === 'className'
-			? 'class'
-			: name
-		: name === 'class'
-			? 'className'
-			: name;
+	if (isSvg) {
+		if (name === 'className') {
+			name = 'class';
+		}
+	}
+	else if (name === 'class') {
+		name = 'className';
+	}
```

Comment:
Original:
```js
l = i ? ("className" === l ? "class" : l) : "class" === l ? "className" : l;
```

Updated:
```js
i ? "className" === l && (l = "class") : "class" === l && (l = "className");
```

The original ternary could be expanded to:
```js
if (isSvg) {
	if (name === 'className') {
		name = 'class';
	}
	else {
		name = name;
	}
}
else {
	if (name === 'class') {
		name = 'className';
	}
	else {
		name = name;
	}
}
```

The new expression simplifies it by removing the redundant `name=name` parts and allowing terser to do its magic.